prod.Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:14 as react-build-step
+
+# Grab needed environment variables from .env.example
+ENV REACT_APP_ENV=production
+
+RUN apt-get update \
+    && apt-get install -y libusb-1.0-0 libusb-1.0-0-dev libudev-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /app
+
+COPY package.json yarn.lock .
+COPY src/logic/contracts/artifacts ./src/logic/contracts/artifacts
+
+RUN yarn install
+
+COPY . .
+
+RUN yarn build
+
+# Second step, we deploy the build
+FROM nginx:alpine
```

Comment:
Should we pin to a specific version instead of pulling the latest i.e. `FROM nginx:1.21.6-alpine`?

Reply:
As `nginx` is very stable and backwards compatible I think this way is alright. It's not as important as it is a `node` version change

Reply:
Either way, I will tag it to `1`

Reply:
Done